src/main/java/pl/orki/hackathon/webapp/band/boundary/dto/BandConverter.java

```diff
@@ -30,7 +33,9 @@ public BandResponseDTO convertToDTO(Band band) {
         Set<City> cities = band.getCities();
         bandDTO.setCitiesIds(getCitiesIds(cities));
         bandDTO.setCities(getCitiesNames(cities));
-        bandDTO.setMusicGenres(band.getMusicGenres().stream().map(MusicGenre::toString).collect(Collectors.toSet()));
+        Set<MusicGenre> musicGenres = band.getMusicGenres();
```

Comment:
Just to make sure: we will receive empty list if there are no music genres?

Reply:
Yes